receiver/kafkareceiver/README.md

```diff
@@ -44,6 +44,8 @@ The following settings can be optionally configured:
 - `group_id` (default = otel-collector): The consumer group that receiver will be consuming messages from
 - `client_id` (default = otel-collector): The consumer client ID that receiver will use
 - `initial_offset` (default = latest): The initial offset to use if no offset was previously committed. Must be `latest` or `earliest`.
+- `session_timeout` (default = `10s`): The request timeout for detecting client failures when using Kafka’s group management facility.
+- `heartbeat_interval` (default = `3s`): The expected time between heartbeats to the consumer coordinator when using Kafka’s group management facilities.
```

Comment:
Tiny nit: One description is using `facilities`, the other is `facility`. It would be good to use one or the other for consistency.

Reply:
Thanks! Should be updated with this feedback now.